raiden/raiden_event_handler.py

```diff
@@ -594,13 +609,16 @@ def handle_contract_send_channelunlock(
                 and gain.from_our_locks == 0
             )
             if not skip_unlock:
-                unlock(
-                    payment_channel=payment_channel,
-                    end_state=restored_channel_state.our_state,
-                    sender=our_address,
-                    receiver=partner_address,
-                    given_block_identifier=channel_unlock_event.triggered_by_block_hash,
-                )
+                try:
+                    unlock(
+                        payment_channel=payment_channel,
+                        end_state=restored_channel_state.our_state,
+                        sender=our_address,
+                        receiver=partner_address,
+                        given_block_identifier=channel_unlock_event.triggered_by_block_hash,
+                    )
+                except InsufficientEth as e:
+                    raise RaidenUnrecoverableError(str(e)) from None
```

Comment:
As discussed, let's not `raise from None`

Reply:
Yes, i changed it to `raise from e` in all places.